ci-operator/README.md

```diff
@@ -0,0 +1,154 @@
+ci-operator
+===========
+
+
+End-to-end tests
+----------------
+
+The preferred way to write e2e tests is using `ci-operator`.  See the
+documentation for details on how to download, build, and execute it:
+
+https://github.com/openshift/ci-operator.git
+
+The first step is writing a configuration file for `ci-operator` for the
+repository.  Many projects already have one, which takes care of most of the CI
+process: downloading the source, building binaries, building RPMs, creating
+images, executing unit tests, etc.  These can be built upon for e2e test,
+requiring little or no modification.
+
+After that, to add an e2e test:
+
+1. Determine the pre-requisites for the test.  Practically, this means choosing
+   the template that is going to be used by `ci-operator`.  This directory
```

Comment:
"template" is also an overloaded term. Make it clear that we're referring to OpenShift Templates (link to the docs describing the concept). And I'm guessing CI Operator uses specific template definitions, so please link to those too.

Also, add a one-line sentence describing the template's purpose (e.g. "This template will drive the test's workflow") just so the reader doesn't have to dive down a rabbit hole to find out what a Template is and what it will do for them.

Reply:
Changed.